tests/unit/test_cli.py

```diff
@@ -956,8 +956,7 @@ def test_a_user_can_successfully_save_sboms_merge_results_to_a_file(
         self,
         sbom_files_to_merge: list[str],
     ) -> None:
-        with tempfile.NamedTemporaryFile(delete=False) as fp:
-            fp.close()
+        with tempfile.NamedTemporaryFile() as fp:
```

Comment:
I introduced a close() call (accompanied by `delete=False`) back in the days of exposing `merge-sboms` as a subcommand to work around some problem with a temporary file, likely a file contention. Back then it was a conscious decision which had to make it into a comment, but apparently did not. Now I wonder what went wrong back then and is fixed now.

Reply:
It is unusual to request a context manager on a temporary file not to be deleted afterwards AND close it right on the next line, I'd be curious to learn what the reason was back then :).